NestingContently.Umbraco.ValueConverters/NestingContently.Umbraco.ValueConverters.csproj

```diff
@@ -15,8 +15,8 @@
 		<NuspecFile>package.nuspec</NuspecFile>
 		<NuspecProperties>id=$(Id);version=$(Version);title=$(Title);description=$(Description);authors=$(Authors);projectUrl=$(PackageProjectUrl);license=$(PackageLicenseExpression);configuration=$(Configuration)</NuspecProperties>
 	</PropertyGroup>
-	<Target Name="UmbPack" AfterTargets="Pack" Condition="'$(Configuration)' == 'Release'">
```

Comment:
This condition was already in place because the `Pack` target is only executed when `GeneratePackageOnBuild` is enabled on `Release` builds.